icons/discordjs/discordjs-original.svg

```diff
@@ -1 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#090a16" d="M0 0h128v128H0Z"/><path fill="#b75cff" d="M45.793 79.102c-.669.285-1.605.67-2.36.908-1.598.503-2.88.707-4.194.906 1.58 5.066 7.769 10.04 14.986 9.57 6.804 0 11.514-3.354 14.02-8.218-7.107-.101-17.46 2.858-22.452-3.166zm64.942.835c-2.485.296-4.26-.204-8.332 1.103-9.857 4.06-19.612.617-22.718-.056-1.02-.106-1.487-1.598-3.58.87l-1.02 1.203c4.817 4.74 11.421 7.46 18.259 7.46 9.207 0 15.432-4.446 17.5-10.486z"/><path fill="#5c6cff" d="M111.257 72.027c-5.414-3.76-6.496-1.042-9.211 1.893.251.642.387 1.372.387 2.22 0 3.963-3.185 6.528-8.857 6.528-4.74 0-9.634-2.331-13.285-5.75l-4.186 4.936c5.668.255 11.406 1.719 17.145 1.492 6.281-.249 14.578-2.237 17.485-3.409l.11.094c.496-1.451.756-2.994.756-4.59 0-1.235-.12-2.368-.344-3.414zm-40.892.082c-3.067-2.34-6.008-1.543-8.827 1.763v.566c0 4.479-3.62 8.11-8.085 8.11a8.075 8.075 0 0 1-6.85-3.797s-.353.156-.81.35c2.113 2.663 4.71 4.04 9.863 4.245 4.197.166 8.394-.569 12.591-1.078 1.402-2.72 2.118-5.911 2.118-9.256z"/><path fill="#5cff9d" d="M70.365 62.503c-2.865-2.521-5.782-3.03-8.827.817v10.552c2.952-.688 5.898-1.387 8.827-1.763zm34.164.766c-4.407-1.929-17.603.001-19.936.885.978.58 1.993 1.076 3.001 1.495l6.915 3.031c3.719 1.55 6.545 2.706 7.537 5.24 2.855-.415 5.316-.983 6.955-1.36l2.256-.533c-.865-4.039-3.316-6.75-6.728-8.758z"/><path fill="#ffdb5c" d="M77.35 52.774c.232 5.603 3.476 9.146 7.26 11.387 2.888.57 5.776 1.064 8.664 1.064 4.032 0 8.059-.96 11.258-1.954a27.04 27.04 0 0 0-3.186-1.584l-6.837-2.876c-3.377-1.376-6.552-2.602-7.662-5.095-6.813-3.459-2.243-5.642-9.496-.942zm-6.985.678c-2.591-1.787-5.514-1.252-8.828 1.619l.001 8.255c2.942-.295 5.885-.687 8.827-.823z"/><path fill="#f79454" d="M80.393 43.72c-1.945 2.337-3.055 5.227-3.055 8.408 0 .218.005.433.013.645 3.15-.031 6.319.318 9.496.944a4.974 4.974 0 0 1-.418-2.055c0-3.73 3.107-5.984 7.925-5.984 4.35 0 7.847 1.632 11.032 4.351l3.382-4.17c-9.884-3.055-18.718-4.976-28.374-2.14Zm-10.028 1.997c-3.646-3.202-6.445-.84-8.828 3.628v5.726c2.943-.456 5.885-1.122 8.828-1.619z"/><path fill="#ff5c5c" d="M61.537 37.83v11.515c2.943-.993 5.885-2.463 8.828-3.629V37.83zm32.816 0c-5.982 0-10.962 2.288-13.959 5.888 4.266.016 8.56.878 12.856 1.391 6.144.734 12.267.75 15.517.75l1.281-1.58c-3.963-3.963-9.713-6.45-15.695-6.45Z"/><path fill="#5865f2" d="M21.085 37.83v38.895H35.81c3.86 0 6.816-.612 9.561-1.996-.005-.096-.005-.193-.005-.29 0-4.48 3.621-8.11 8.087-8.11.132 0 .263.003.394.01 1.401-2.852 1.966-5.68 1.966-9.395 0-3.834-.722-7.17-2.167-10.058-1.5-2.834-3.667-5.056-6.556-6.668-2.89-1.611-6.39-2.388-10.558-2.388zm9.335 8.334h5.612c3.279 0 5.779.945 7.446 2.89 1.667 1.944 2.5 4.611 2.5 8.056 0 3.5-.89 6.224-2.612 8.224-1.722 2.056-4.278 3.056-7.668 3.056H30.42zM53.476 69.33a5.11 5.11 0 1 0 0 10.218 5.11 5.11 0 0 0 0-10.218z"/></svg>
```

Comment:
Can we remove the background from this one, or would it look weird?

Reply:
I think is good the way it is, but it'll be nice to remove the background, because I can also make it and `plain` fit the view box. So, I'll remove and resize it. 🙂